lib/internal/cluster/master.js

```diff
@@ -102,11 +102,14 @@ function createWorkerProcess(id, env) {
   const workerEnv = util._extend({}, process.env);
   const execArgv = cluster.settings.execArgv.slice();
   const debugArgRegex = /--inspect(?:-brk|-port)?|--debug-port/;
+  const nodeOptions = process.env.NODE_OPTIONS ?
+    process.env.NODE_OPTIONS.split(' ') : [];
```

Comment:
There's no need to split `NODE_OPTIONS`. Later on `match()` can be called on the whole string.

Reply:
@richardlau I have removed the split and doing a direct match. Please review again.